src/test/java/SetTest.java

```diff
@@ -0,0 +1,46 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("Set Collection에 대한 학습 테스트")
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set의 크기를 확인한다.")
+    @Test
+    void size() {
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```
으로도 작성할 수 있을 것 같아요...!